charter-template.html

```diff
@@ -334,10 +334,10 @@ <h2 id="participation">
           To be successful, this <i class="todo">(Working|Interest)</i> Group is expected to have 6 or more active participants for its duration, including representatives from the key implementors of this specification, and active Editors and Test Leads for each specification. The Chairs, specification Editors, and Test Leads are expected to contribute half of a working day per week towards the <i class="todo">(Working|Interest)</i> Group. There is no minimum requirement for other Participants.
         </p>
         <p>
-          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>.
+          The group encourages questions, comments and issues on its public mailing lists and document repositories, as described in <a href='#communication'>Communication</a>. The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
         </p>
         <p>
-          The group also welcomes non-Members to contribute technical submissions for consideration upon their agreement to the terms of the <a href="https://www.w3.org/policies/patent-policy/">W3C Patent Policy</a>.
+          The Chairs should periodically look through the non-Members who have contributed to the Working Group <i class="todo">or the [name of an associated Community Group, if any]</i> and consider whether each one should be invited to participate as an <a href="https://www.w3.org/invited-experts/">Invited Expert</a>. If a non-Member contributor would like to participate in meetings, they are encouraged to <i class="todo">[update this link] <a href="https://www.w3.org/groups/wg/TODO/instructions/">apply to be an Invited Expert</a></i>.
```

Comment:
This takes "assurances in place" into account as mentioned in https://github.com/w3c/cg-council/issues/18#issuecomment-2374786608 but a stronger language may be necessary. It also acknowledges that the charter relies heavily on significant contributions from those original contributors.


```suggestion
          The Chairs and the W3C Team contact are expected to invite non-Members who have contributed to the input documents under <a href="#deliverables">deliverables</a> to participate as an <a href="https://www.w3.org/invited-experts/">Invited Expert</a>. The Chairs should periodically look through the non-Members who have contributed to the Working Group <i class="todo">or the [name of an associated Community Group, if any]</i> and consider whether each one should be invited to participate as an <a href="https://www.w3.org/invited-experts/">Invited Expert</a>. If a non-Member contributor would like to participate in meetings, they are encouraged to <i class="todo">[update this link] <a href="https://www.w3.org/groups/wg/TODO/instructions/">apply to be an Invited Expert</a></i>.
```

Reply:
I don't think we can set an invitation expectation on Chairs and Team Contacts. There are other factors to take into consideration, such as the contributor affiliation.

Reply:
Fair enough but then how can one set / expect / suggest any level of assurance or be more inviting?

Wouldn't any factors that should be taken into account already be part of the invitation process or criteria?

Reply:
To clarify, the main point of my suggestion is to better acknowledge and elevate significant contributors of original work. After all, without these contributors incubating the work, neither the charter nor the Group would exist in the first place. Would it?

I'm not fixed on the word "expectation". If another word or phrasing better captures the essence, that would be fine.

I'd like to comment on the following sentence regarding contributions to ongoing work:

>The Chairs should periodically look through the non-Members [...] and consider whether each one should be invited.

This statement also sets an expectation. If the distinction between the first and second statement is about ensuring they are invited vs. considering inviting them, then the first sentence could at least use "consider" as well. However, I find that a bit awkward. Why would the Chairs or Team Contacts consider (rather than directly invite) those who have been significant contributors to the foundational documents?

Reply:
"Expected to invite non-Members" sets up too strong an expectation. I don't want "wy am I not an IE yet" mails from a bot that made five spelling error PRs, for example.

Reply:
So, I plan to merge this PR, but without [this suggestion](https://github.com/w3c/charter-drafts/pull/598/files#r1776731405).